src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisitionExtensionOptions.cs

```diff
@@ -62,26 +41,32 @@ internal void InvokeOnBuildConfidentialClientApplication(ConfidentialClientAppli
         }
 
         /// <summary>
-        /// Invoke the OnBeforeTokenAcquisitionForApp event.
+        /// Invoke the OnAfterTokenAcquisition event.
         /// </summary>
-        internal void InvokeOnBeforeTokenAcquisitionForApp(AcquireTokenForClientParameterBuilder builder,
-    AcquireTokenOptions? acquireTokenOptions)
+        internal void InvokeOnAfterTokenAcquisition(AuthenticationResult result,
```

Comment:
Why invert both methods?

Reply:
Those 2 methods are not used yet

Reply:
It's fewer changes if you keep the previous ordering

Reply:
I was trying to consolidate them so I can put everything under one preprocessor tag. I can switch it back, either way works.

Reply:
This is just to not lose the code apparently. @jmprieur where do you want to put this code? it's not wired up right now and would prefer to not merge commented code